build/tools/templates/RS.References/RS.References.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
```

Comment:
I made a contending change in #598 to use `src/Directory.Build.props` instead of the import to this file.

Reply:
That's src though? This project gets copied over to .w/RS.References/RS.REferences.csproj

Reply:
Yup, I copy directory.build.props too. https://github.com/aspnet/Universe/pull/598/files#diff-20ec4e3e95e20c986d83f609fb34ba1aR34